Packs/QualysFIM/ReleaseNotes/1_0_34.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Qualys FIM
+
+Update all usages of the unsupported API V1 to the new API V2 and V3. The changes affect the integration instance "Test" button and the ***qualys-fim-event-get*** command.
```

Comment:
```suggestion
Added support for QualysFIM API v2 & v3, changes apply the instance test button and the ***qualys-fim-event-get*** command.
```

Reply:
Fixed in commit c264c71294807bac2a29bc298d788fb475d0dd1b